python/src/uagents/registration.py

```diff
@@ -293,3 +293,40 @@ async def register(
         except Exception as e:
             self._logger.error(f"Failed to register on Almanac contract: {e}")
             raise
+
+
+class AgentStatusUpdate(BaseModel):
```

Comment:
This is the same structure as the `AgentRegistrationAttestation` so we should make both inherit from the same model.